internal/controller/ingress/tcpedge_controller.go

```diff
@@ -388,26 +388,25 @@ func (r *TCPEdgeReconciler) listTCPEdgesForIPPolicy(ctx context.Context, obj cli
 
 func (r *TCPEdgeReconciler) updatePolicyModule(ctx context.Context, edge *ingressv1alpha1.TCPEdge, remoteEdge *ngrok.TCPEdge) error {
 	policy := edge.Spec.Policy
-	client := r.NgrokClientset.EdgeModules().TCP().Policy()
-
-	endpointPolicy := policy.ToNgrok()
+	client := r.NgrokClientset.EdgeModules().TCP().RawPolicy()
 
 	// Early return if nothing to be done
-	if endpointPolicy == nil {
+	if policy == nil {
 		if remoteEdge.Policy == nil {
-			r.Log.Info("Module matches desired state, skipping update", "module", "Policy", "comparison", routeModuleComparisonBothNil)
+			r.Log.Info("Module matches desired state, skipping update", "module", "TrafficPolicy", "comparison", routeModuleComparisonBothNil)
```

Comment:
Other two specs don't update this, and we aren't updating the field name for backwards compatibility, so maybe change these 3 `TrafficPolicy` back to `Policy`?